recipes/fruit/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "3.4.0":
+    url: "https://github.com/google/fruit/archive/v3.4.0.tar.gz"
+    sha256: "0f3793ee5e437437c3d6360a037866429a7f1975451fd60d740f9d2023e92034"
+patches:
+  "3.4.0":
+    - patch_file: "patches/0001-conan-cmake.patch"
```

Comment:
Please improve the name of this patch. What is it doing?

Reply:
Yes. I realize that it's an unclear name for what I'm doing.

On the first commit 1c9f5e0a15fc196459b72900e1ed08e71ed96746, the patch was split.
Modified CMakeLists.txt so that fruit works with CMake multi-config generators. Then the patches started to conflict. Patch conflicts were inevitable, so I ended up in one patch.

The work of the patch is enumerated as follows:

* Allow CMake multi-config generators.
* Remove extra CMake targets (examples, tests and extras).
* Set CMake RUNTIME DESTINATION

If all of this work was included in the name, it would be very long (I'm not a native speaker, so I couldn't come up with a good name).

Should I include all work in the patch name, even if it's a long name?

Reply:
Thanks for explaining and thanks for upstreaming parts of the patch (https://github.com/google/fruit/pull/104)

In that case I think it is fine for now

Reply:
Oh, you found my pull request.

The pull request was created as a backport during the creation of the conan recipe.
The changes did not match because the source state was different between v3.4.0 and master.

I will have to create a new patch with the next fruit version upgrade.

Reply:
@tt4g If the changes for having a static fruit are small, a patch can always do the trick.
Alternatively, if a v3.4.1 (with static support) arrives soon, you can also just skip 3.4.0.

Reply:
@madebr Sorry. I overlooked your comment.

fruit's CMake was based on version 2.8 and does not support CMake multi-configuration generators.
The patch had to make many changes to support CMake multi-config generators.

I wanted to modernize CMakeLists.txt and send a pull request, but it seemed very difficult because fruit's CMake had a lot of tool support such as automatic generation of pytest, vagrant support, etc.

Reply:
In addition, the fruit maintainers had stated in 2019 that they were busy.
I think It is unlikely that the next version will be released soon.

Reply:
If that's the situation, that will do.
Thanks for the update!